user-guide/enterprise-auth/intro.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Using Enterprise Authentication
+description: Learn how to connect to your enterprise authentication provider, and how to leverage data from that provider in your apps.
+position: 1
+publish: true
+slug: enterprise-auth
+---
+
+# Using Enterprise Authentication
+
+NativeScript Sidekick lets you connect to existing enterprise identity and single sign-on solutions. This powerful functionality allows application developers to leverage enterprise user accounts, and to do so while avoiding the complexity of integrating these protocols into mobile apps.
+
+Let’s look at how it works.
+
+* [Step 1. Start you app](#step-1)
```

Comment:
Fixed a small typo here. It was (#step-2) instead of (#step-1).

Reply:
Just noticing it should also say Start _your_ app :)